reference-architecture/rhv-ansible/example/nsupdate-clean.txt

```diff
@@ -0,0 +1,119 @@
+server localhost
```

Comment:
is this huge file really needed?

Reply:
It's really needed for my env, and could provide a starting point for others.